README.md

```diff
@@ -8,8 +8,8 @@ Sometimes your rows might come in a nested format or it might have a representat
 import * as resolve from 'table-resolver';
 
 // Or you can cherry-pick
-import { index } from 'table-resolver';
-import { index as resolveIndex } from 'table-resolver';
+import { nested } from 'table-resolver';
+import { nested as resolveNested } from 'table-resolver';
 ```
 
 ## API
```

Comment:
I have to adapt some other code to work without it. I'll most likely merge this tomorrow and do the necessary work. Thanks. 👍